definitions/divorce/json/CaseEventToFields/CaseEventToFields-nonprod.json

```diff
@@ -192,8 +192,8 @@
     "LiveFrom": "20/09/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "EnterAOSOffline_CoRespondent",
-    "CaseFieldID": "CoRespStatementOfTruth",
-    "PageFieldDisplayOrder": 9,
+    "CaseFieldID": "CoRespondentSolicitorRepresented",
+    "PageFieldDisplayOrder": 13,
```

Comment:
Might be good to start from 9. So it is clearer that this comes straight after "CoRespCostsReason"

Reply:
Ok